src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -160,12 +165,14 @@ object SeedJointSelector {
       } else {
         // Joint meets criteria
         val remainingVolume = jointOption.get
-        if (remainingVolume.volume <= volumePerPiece) {
+        val nonRedundantVolume = Block(remainingVolume.center, remainingVolume.nonRedundantFaces)
```

Comment:
Unfortunately, we have to eliminate non-redundant faces every time we want to find the volume of a block - simplex integration needs oriented vertices and to find the vertices all redundant faces need to be removed.